src/AutoMapper/ConfigurationValidator.cs

```diff
@@ -11,10 +11,12 @@ namespace AutoMapper
     public class ConfigurationValidator
     {
         private readonly IConfigurationProvider _config;
+        private readonly Action<ValidationContext> _validator;
 
-        public ConfigurationValidator(IConfigurationProvider config)
+        public ConfigurationValidator(IConfigurationProvider config, Action<ValidationContext> validator)
```

Comment:
OK, so you want a list of actions? I don't know if that's useful, but for consistency.